pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -23,42 +23,42 @@ option java_package = "org.apache.pulsar.common.api.proto";
 option optimize_for = LITE_RUNTIME;
 
 message Schema {
-	enum Type {
```

Comment:
Seems most of the changes in this file is the code alignment change. I would suggest keep the original format, so it would be easy to find what is really changed for this PR.

Reply:
Sorry for the inconvenience to your review.  All I added commands are under the comments called the transaction related.

I updated the alignments because there are two types of alignments in this file. I replace all the tab with 4 spaces. And I think it is easier to keep consistent when we update this file in the future.

Reply:
OK. It is better to do the alignments in a separate PR.

Reply:
Thanks for your reminder. I will pay more attention next time.